### PR TITLE
Adding docs for syncable exports

### PR DESCRIPTION
**Pull request by parthaa**
* [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Comment by parthaa** [comment]
@Lennonka can ya review ?

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 1]
```diff
@@ -0,0 +1,83 @@
+[id="Exporting_a_Content_View_Version_Syncable_{context}"]
+= Exporting a Content View Version in a Syncable Format
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:2]: Please adjust the _anchor and file name_ to match the title aka `[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]`

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+You can export a version of a Content View to an archive file from {ProjectServer} in a syncable format.
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing  {ProjectServer} or in another {ProjectServer} organization.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:7]:
```suggestion
You can then serve the generated content via a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}
+* Update your CDN configuration to *Custom CDN*
+* Set the CDN URL to point to the web server
+* Optionally set a SSL CA Credential if the web server requires it
+* Enable the Repository
+* Synchronize the Repository.
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization via content import.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:19]:
```suggestion
You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization using content import.
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
+endif::[]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:21]:
```suggestion
For more information, see xref:Importing_a_Content_View_Version_{context}[].
endif::[]
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+You can export the following content in a syncable format from {ProjectServer}:
+* RPM repositories
+* Kickstart repositories
+* File repositories
+You cannot export Ansible and Docker content.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:27]:
```suggestion

You cannot export Ansible and Docker content.
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization via content import.
+On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
+endif::[]
+
+You can export the following content in a syncable format from {ProjectServer}:
+* RPM repositories
+* Kickstart repositories
+* File repositories
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:26]:
```suggestion
* File repositories
* Kickstart repositories
* RPM repositories
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories
+endif::[]
+
+The export contains the following data:
+
+* Directories containing the rpms, listing files and metadata of each repository in yum format that could then be used synchronize in the importing {ProjectServer}.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:35]:
```suggestion
The export of RPM repositories contains directories containing the packages, listing files, and metadata of each repository in Yum format.
You can use this information to import the content view version into {ProjectServer}.
```

Does this also apply to Kickstart and File repositories?

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+endif::[]
+
+You can export the following content in a syncable format from {ProjectServer}:
+* RPM repositories
+* Kickstart repositories
+* File repositories
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:30]:
```suggestion
You cannot export DEB repositories.
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+endif::[]
+
+The export contains the following data:
+
+* Directories containing the rpms, listing files and metadata of each repository in yum format that could then be used synchronize in the importing {ProjectServer}.
+
+.Prerequisites
+
+To export a Content View, ensure that {ProjectServer} where you want to export meets the following conditions:
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:40]:
```suggestion
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+For more information, see xref:Download_Policies_Overview_{context}[].
+* Ensure that you synchronize Products that you export to the required date.
+* Ensure that the user exporting the content has the `Content Exporter` role.
+
+.To Export a Content View Version
+. List versions of the Content View that are available for export:
++
+[subs="+quotes"]
+----
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:51]:
```suggestion
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+--organization="_My_Organization_"
+
+---|----------|---------|-------------|-----------------------
+ID | NAME     | VERSION | DESCRIPTION | LIFECYCLE ENVIRONMENTS
+---|----------|---------|-------------|-----------------------
+5  | view 3.0 | 3.0     |             | Library
+4  | view 2.0 | 2.0     |             |
+3  | view 1.0 | 1.0     |             |
+---|----------|---------|-------------|----------------------
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:63]:
```suggestion
```

We generally avoid interactive shell sessions throughout the docs.

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+----
+
+.Export a Content View version
+. Get the version number of desired version.
+The following example targets version `1.0` for export.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:68]:
```suggestion
The following example targets version `1.0` for export:
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+.Prerequisites
+
+To export a Content View, ensure that {ProjectServer} where you want to export meets the following conditions:
+
+* Ensure that you set download policy to *Immediate* for all repositories within the Content View you export.
+For more information, see xref:Download_Policies_Overview_{context}[].
+* Ensure that you synchronize Products that you export to the required date.
+* Ensure that the user exporting the content has the `Content Exporter` role.
+
+.To Export a Content View Version
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:46]: Maybe you can move the into the procedure below, add a `.Procedure` heading and make the step optional?

File: guides/common/modules/proc_exporting-a-repository-syncable.adoc [hunk 14]
```diff
@@ -0,0 +1,66 @@
+[id="Exporting_a_Repository_Syncable_{context}"]
+= Exporting a Repository in a Syncable Format
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-syncable.adoc:2]: As above: Please ensure that the title, anchor, and file name match.

File: guides/common/modules/proc_exporting-a-repository-syncable.adoc [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+
+You can export the content of a repository in the Library environment of an organization from {ProjectServer} in a syncable format.
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing  {ProjectServer} or in another {ProjectServer} organization.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-syncable.adoc:6]:
```suggestion
You can then serve the generated content using a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
```

File: guides/common/modules/proc_exporting-a-repository-syncable.adoc [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}
+* Update your CDN configuration to *Custom CDN*
+* Set the CDN URL to point to the web server
+* Optionally set a SSL CA Credential if the web server requires it
+* Enable the Repository
+* Synchronize the Repository.
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization via content import.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-syncable.adoc:18]:
```suggestion
You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization via content import.
```

Please see my suggestion above.

File: guides/common/modules/proc_exporting-a-repository-syncable.adoc [hunk 17]
```diff
@@ -0,0 +1,66 @@
...
+from {ProjectServer}.
+
+The export contains the following data:
+
+* Directories containing the rpms, listing files and metadata of the repository in yum format that could then be used synchronize in the importing {ProjectServer}.
+
+.Prerequisites
+
+To export the contents of a repository, ensure that {ProjectServer} from which you want to export, meets the following conditions:
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-syncable.adoc:39]:
```suggestion
```

File: guides/common/modules/proc_exporting-a-repository-syncable.adoc [hunk 18]
```diff
@@ -0,0 +1,66 @@
...
+ifndef::satellite[]
+You cannot export DEB repositories
+endif::[]
+from {ProjectServer}.
+
+The export contains the following data:
+
+* Directories containing the rpms, listing files and metadata of the repository in yum format that could then be used synchronize in the importing {ProjectServer}.
+
+.Prerequisites
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-syncable.adoc:36]:
```suggestion
.Prerequisite
```

File: guides/common/modules/proc_exporting-a-repository-syncable.adoc [hunk 19]
```diff
@@ -0,0 +1,66 @@
...
+
+To export the contents of a repository, ensure that {ProjectServer} from which you want to export, meets the following conditions:
+
+* Ensure that you set download policy to *Immediate* for the repository within the Library lifecycle environment you export.
+For more information, see xref:Download_Policies_Overview_{context}[].
+
+.Export a Repository
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-syncable.adoc:43]:
```suggestion
.Procedure
```

File: guides/common/modules/proc_exporting-a-repository-syncable.adoc [hunk 20]
```diff
@@ -0,0 +1,66 @@
...
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-syncable.adoc:44]:
```suggestion
```

File: guides/common/modules/proc_exporting-a-repository-syncable.adoc [hunk 21]
```diff
@@ -0,0 +1,66 @@
...
+. Use the repository name or ID to export.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-syncable.adoc:45]:
```suggestion
. Export a repository using the repository name or ID:
```

or maybe "based on ..."?

File: guides/common/modules/proc_exporting-a-repository-syncable.adoc [hunk 22]
```diff
@@ -0,0 +1,66 @@
...
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-export complete repository \
+ --organization="_My_Organization_" \
+ --product="_My_Product_" \
+ --name="_My_Repository_"\
+ --format=syncable
+----
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-syncable.adoc:55]:
```suggestion
```

File: guides/common/modules/proc_exporting-a-repository-syncable.adoc [hunk 23]
```diff
@@ -0,0 +1,66 @@
...
+. Optional: Verify that the exported content is located in the export directory:
++
+[options="nowrap" subs="+quotes"]
+----
+# ls -lh /var/lib/pulp/exports/_My_Organization_/Export-_My_Repository_/1.0/2021-03-02T03-35-24-00-00
+total 0
+drwxr-xr-x. 3 pulp pulp 33 Aug 26 09:16 content
+
+# du -sh /var/lib/pulp/exports/_My_Organization_/Export-_My_Repository_/1.0/2021-03-02T03-35-24-00-00
+439M
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-syncable.adoc:65]:
```suggestion
```

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 24]
```diff
@@ -0,0 +1,63 @@
+[id="Exporting_the_Library_Environment_Syncable{context}"]
+= Exporting the Library Environment in a Syncable Format
+
+You can export contents of all Yum repositories in the Library environment of an organization to an archive file from {ProjectServer} in syncable format.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:4]: Does this only apply to Yum content?

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 25]
```diff
@@ -0,0 +1,63 @@
...
+endif::[]
+
+The export contains the following data:
+
+* Directories containing the rpms, listing files and metadata of each repository in yum format that could then be used synchronize in the importing {ProjectServer}.
+
+.Prerequisites
+
+To export the contents of the Library lifecycle environment of the organization, ensure that {ProjectServer} where you want to export meets the following conditions:
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:41]:
```suggestion
```

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 26]
```diff
@@ -0,0 +1,63 @@
...
+* Ensure that you set download policy to *Immediate* for all repositories within the Library lifecycle environment you export.
+For more information, see xref:Download_Policies_Overview_{context}[].
+* Ensure that you synchronize Products that you export to the required date.
+* Ensure that the user exporting the content has the `Content Exporter` role.
+
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:47]:
```suggestion
```

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 27]
```diff
@@ -0,0 +1,63 @@
...
+.Export the Library Content of an Organization
+. Use the organization name or ID to export.
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-export complete library \
+--organization="_My_Organization_" \
+--format=syncable
+----
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:57]:
```suggestion
```

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 28]
```diff
@@ -0,0 +1,63 @@
...
+To export the contents of the Library lifecycle environment of the organization, ensure that {ProjectServer} where you want to export meets the following conditions:
+
+* Ensure that you set download policy to *Immediate* for all repositories within the Library lifecycle environment you export.
+For more information, see xref:Download_Policies_Overview_{context}[].
+* Ensure that you synchronize Products that you export to the required date.
+* Ensure that the user exporting the content has the `Content Exporter` role.
+
+
+.Export the Library Content of an Organization
+. Use the organization name or ID to export.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:49]: See above.

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 29]
```diff
@@ -0,0 +1,63 @@
...
+
+To export the contents of the Library lifecycle environment of the organization, ensure that {ProjectServer} where you want to export meets the following conditions:
+
+* Ensure that you set download policy to *Immediate* for all repositories within the Library lifecycle environment you export.
+For more information, see xref:Download_Policies_Overview_{context}[].
+* Ensure that you synchronize Products that you export to the required date.
+* Ensure that the user exporting the content has the `Content Exporter` role.
+
+
+.Export the Library Content of an Organization
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:48]:
```suggestion
.Procedure
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
Thanks for your PR! I made some suggestions. Please also avoid using `via` throughout the docs.

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 30]
```diff
@@ -0,0 +1,83 @@
+[id="Exporting_a_Content_View_Version_Syncable_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to an archive file from {ProjectServer} in a syncable format.
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing  {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead on the importing {ProjectServer} you would need to
```
Comment (Lennonka) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:9]:
```suggestion
You cannot directly import Syncable Format exports. Instead, on the importing {ProjectServer} you must:
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 31]
```diff
@@ -0,0 +1,83 @@
+[id="Exporting_a_Content_View_Version_Syncable_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to an archive file from {ProjectServer} in a syncable format.
```
Comment (Lennonka) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:4]: Why? When do I have to use the syncable format?
I wonder if we should describe the bigger picture of this scenario somewhere in the introduction.

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc (hunk 4 above)
Comment (Lennonka) [reply guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:21]: FYI, there will be a special module for syncable import.

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc (hunk 7 above)
Comment (Lennonka) [reply guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:35]: And I'd format `listing` as code to make it look like a literal.

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc (hunk 31 above)
Comment (Lennonka) [reply guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:4]: Maybe we could elaborate on how the content is syncable after it's exported?
Also, this kind of export does not result in an "archive file" :)

So perhaps something like: "You can export a version... to a format that can be used to synchronize the content from a local custom CDN?"

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc (hunk 7 above)
Comment (parthaa) [reply guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:37]: > Does this also apply to Kickstart and File repositories?

Listing files are generated for yum/kickstart only. In fact they are generated just for RH content not custom.

**Comment by ekohl** [comment]
@parthaa in the files overview you can apply multiple suggestions at once. This then only generates a single commit instead of n, which also means just a single email. This is kinder on people's inboxes and on CI resources.

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc (hunk 11 above)
Comment (parthaa) [reply guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:64]: hmm i saw this kinda output here -> https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_exporting-a-content-view-version.adoc

I am open to removing from both if you are ok with it.

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc (hunk 24 above)
Comment (parthaa) [reply guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:4]: Yum, KS and File

**Comment by parthaa** [comment]
> @parthaa in the files overview you can apply multiple suggestions at once. This then only generates a single commit instead of n, which also means just a single email. This is kinder on people's inboxes and on CI resources.

Thanks. Will keep that in mind in the future

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc (hunk 11 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:62]: Removing both sounds like a good idea to me.

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 32]
```diff
@@ -0,0 +1,82 @@
+[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to a syncabe format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
```
Comment (Lennonka) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:4]:
```suggestion
You can export a version of a Content View to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 33]
```diff
@@ -0,0 +1,82 @@
...
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
```
Comment (Lennonka) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:7]: I wonder if we should keep this sentence for more builds than just Satellite. This feature is available in community products as well, isn't it?

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 34]
```diff
@@ -0,0 +1,82 @@
...
+
+You cannot directly import Syncable Format exports. Instead, on the importing {ProjectServer} you must:
+
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}
+* Update your CDN configuration to *Custom CDN*
+* Set the CDN URL to point to the web server
+* Optionally set a SSL CA Credential if the web server requires it
+* Enable the Repository
+* Synchronize the Repository.
+endif::[]
```
Comment (Lennonka) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:17]: Consider this a non-blocker, but I think we're under-selling the feature here. We should at least have a module that says "this is how you create your custom CDN before you configure Satellite Server to sync the content from it".

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 35]
```diff
@@ -0,0 +1,82 @@
...
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization using content import.
+On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
+For more information, see xref:Importing_a_Content_View_Version_{context}[].
+endif::[]
+
+You can export the following content in a syncable format from {ProjectServer}:
+* File repositories
```
Comment (Lennonka) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:26]:
```suggestion
You can export the following content in a syncable format from {ProjectServer}:

* File repositories
```
For proper list rendering.

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc [hunk 36]
```diff
@@ -0,0 +1,82 @@
...
+
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories.
+endif::[]
+
+The export contains the following data:
+
+* Directories containing the rpms, *listing* files and metadata of each repository in yum format that could then be used synchronize in the importing {ProjectServer}.
```
Comment (Lennonka) [inline guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:38]:
```suggestion
* Directories containing the RPMs, *listing* files and metadata of each repository in yum format that could then be used synchronize in the importing {ProjectServer}.
```

File: guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc (hunk 34 above)
Comment (Lennonka) [reply guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc:17]: I'll look into it once we will have "Syncable Imports" available in Sat.

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 37]
```diff
@@ -0,0 +1,61 @@
+[id="Exporting_the_Library_Environment_in_a_Syncable_Format_{context}"]
+= Exporting the Library Environment in a Syncable Format
+
+You can export contents of all Yum repositories in the Library environment of an organization to a syncabe format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
```
Comment (jeremylenz) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:4]:
```suggestion
You can export contents of all Yum repositories in the Library environment of an organization to a syncable format that you can use to create a custom CDN.
```

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 38]
```diff
@@ -0,0 +1,61 @@
...
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing  {ProjectServer} or in another {ProjectServer} organization.
```
Comment (jeremylenz) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:7]:
```suggestion
You can then serve the generated content over HTTP or HTTPS from a local webserver on the importing  {ProjectServer} or in another {ProjectServer} organization.
```

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 39]
```diff
@@ -0,0 +1,61 @@
...
+
+You cannot directly import Syncable Format exports. Instead on the importing {ProjectServer} you would need to
```
Comment (jeremylenz) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:9]:
```suggestion
You cannot directly import Syncable Format exports. Instead, on the importing {ProjectServer}, perform the following steps:
```

Shouldn't this be a "procedure" or something?  (Docs experts will know better..)

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 40]
```diff
@@ -0,0 +1,61 @@
...
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}
+* Update your CDN configuration to *Custom CDN*
+* Set the CDN URL to point to the web server
+* Optionally set a SSL CA Credential if the web server requires it
+* Enable the Repository
+* Synchronize the Repository.
```
Comment (jeremylenz) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:15]: Aren't some of these steps done on the upstream server, and others on the downstream? Seems like it's glossing over this.

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 41]
```diff
@@ -0,0 +1,61 @@
...
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repositories in another {ProjectServer} or in another {ProjectServer} organization via content import.
+On import the products and repositories are mirrored on your downstream {ProjectServer}.
```
Comment (jeremylenz) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:20]: Does this apply if we only have syncable exports and not syncable imports?

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 42]
```diff
@@ -0,0 +1,61 @@
...
+
+
+You can export the following content in a syncable format from {ProjectServer}:
+* RPM repositories
+* Kickstart repositories
+* File repositories
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories
```
Comment (jeremylenz) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:31]:
```suggestion
You cannot export DEB repositories.
```

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 43]
```diff
@@ -0,0 +1,61 @@
...
+
+The export contains the following data:
+
+* Directories containing the rpms, *listing* files and metadata of each repository in yum format that could then be used synchronize in the importing {ProjectServer}.
+
+.Prerequisites
+
+To export the contents of the Library lifecycle environment of the organization, ensure that {ProjectServer} where you want to export meets the following conditions:
+
+* Ensure that you set download policy to *Immediate* for all repositories within the Library lifecycle environment you export.
```
Comment (jeremylenz) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:42]:
```suggestion
* Ensure that you set download policy to *Immediate* for all repositories within the Library environment you export.
```

File: guides/common/modules/proc_exporting-the-library-environment-syncable.adoc [hunk 44]
```diff
@@ -0,0 +1,61 @@
...
+.Procedure
+. Use the organization name or ID to export.
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-export complete library \
+--organization="_My_Organization_" \
+--format=syncable
+----
+. Optional: Verify that the exported content is located in the export directory:
```
Comment (jeremylenz) [inline guides/common/modules/proc_exporting-the-library-environment-syncable.adoc:56]:
```suggestion
.Verification
```
This is how we do this now, right? :smile:

**Review by jeremylenz — COMMENTED** [review]
thanks @parthaa!

left some comments/questions below

**Comment by Lennonka** [comment]
FYI, @bangelic opened a new PR for this: #1737 
I suggest that we close this PR and Brian, please take comments from this one into consideration. We're missing the bigger picture here (conceptual information) and there should probably be one more procedure on creating a custom CDN using syncable exports.

**Pull request closed without merging**